consumer/src/main/java/dk/dbc/pgqueue/consumer/QueueWorker.java

```diff
@@ -117,6 +118,7 @@
         private ExecutorService executor;
         private MetricAbstraction metricsAbstraction;
         private DeduplicateAbstraction<T> deduplicateAbstraction;
+        private Boolean includePostponedInDeduplication;
```

Comment:
Almindelig boolean? er koblet med deduplicateAbstraction - setOrFail til felje på dén og aldrig kommet til include*
Så kan man slippe for at have x != null && x hvor der skal vælges sql stmt